conans/client/tools/win.py

```diff
@@ -575,7 +575,7 @@ def unix_path(path, path_flavor=None):
     return None
 
 
-def run_in_windows_bash(conanfile, bashcmd, cwd=None, subsystem=None, msys_mingw=True, env=None,
+def run_in_windows_bash(conanfile, bashcmd, output=True, cwd=None, subsystem=None, msys_mingw=True, env=None,
```

Comment:
Better add it as last argument, just in case someone is using ``tools.run_in_windows_bash`` with positional args.

Reply:
You're right, I'll make this change.